### ISSUE TITLE
License

### ISSUE THREAD

**Issue by AMGold99** [AMGold99/ricardian#4]
What license should we use? I have used the GPL3 license for building R packages before, but I know the MIT license could also be relevant. How do we decide which to use/whether it's necessary?

- [ ] Decide whether license is necessary, choose a license

**Comment by binders1** [comment]
I don't think it matters too much. Nolte uses a CC0 1.0 Universal (CC0 1.0) Public Domain Dedication (https://creativecommons.org/publicdomain/zero/1.0/) license for the limited data he makes available on Dryad. Maybe the MIT license or GPL3 are more appropriate for software specifically.
